### PR TITLE
Update checkout version

### PR DESCRIPTION
**Pull request by vps-victor**
### Why:

Closes https://github.com/skills/github-pages/issues/24

### What's being changed:

The Checkout version of the workflow

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: .github/workflows/0-start.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
     steps:
       # We'll need to check out the repository so that we can edit the README
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.3.0
```
Comment (rsese) [inline .github/workflows/0-start.yml:39]:
```suggestion
        uses: actions/checkout@v3
```

Can you update the changes to ☝️ so we follow the other skills and pin to the major version e.g. https://github.com/skills/continuous-integration/pull/19?

**Review by rsese — CHANGES_REQUESTED** [review]
Thank you, just one request to update to `v3` and we should be good!

File: .github/workflows/1-enable-github-pages.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
     steps:
       # We'll need to check out the repository so that we can edit the README
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.3.0
```
Comment (sinsukehlab) [inline .github/workflows/1-enable-github-pages.yml:35]: The same rule goes.
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/5-merge-your-pull-request.yml [hunk 3]
```diff
@@ -34,7 +34,7 @@ jobs:
     steps:
       # We'll need to check out the repository so that we can edit the README
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.3.0
```
Comment (vps-victor) [inline .github/workflows/5-merge-your-pull-request.yml:37]:
```suggestion
        uses: actions/checkout@v3
```

**Review by sinsukehlab — APPROVED** [review]

**Pull request merged**